README.md

```diff
@@ -16,25 +16,39 @@ Or
 
 
 ## Run with Docker
-
+- If you are on your local machine, we recommend that you run this container using Docker.  
+- We have tested running this container using Docker on local mac machines and also on a Pawsey Nimbus instance.
+  
+### On a local (mac) machine
 To run this container:
-
 - Start docker
 - Mount a directory housing your data (specify which paths you need to mount), and run the RStudio server instance using the following command: 
 
 ```bash 
 docker run -p 8787:8787 \
-    -e PASSWORD=yourpassword \
+    -e PASSWORD='yourpassword' \
     -v /path/on/host:/home/rstudio sydneyinformaticshub/rnaseq-rstudio:4.1.0
 ```
-
 * `-p 8787:8787` maps port 8787 in the container to 8787 on your host
 * `-e PASSWORD=yourpassword` sets the password you'll need to use to open this in your browser
 
-### Run RStudio in your browser
+**Run RStudio in your browser**  
 
-- Open up a browser window at http://localhost:8787/ 
-- Enter the username: rstudio
+You can now navigate a local web browser at http://localhost:8787/
+
+- Enter the username: `rstudio`
+- Enter the password you gave it e.g `yourpassword` as seen in the above command
+
+### On Pawsey Nimbus instance
+Assuming you are running Docker on a headless remote server, you must ensure you connect to the host with port forwarding enabled if you wish to view the RStudio session on your local machine.
```

Comment:
What does it mean to run docker on a headless remote server??? Simplify this for your audience.

Reply:
Assuming you are running Docker on a remote server without a graphical user interface (GUI)

changed